### DISCUSSION TITLE
Add to calendar not working on iphone- safari

### DISCUSSION THREAD

**Discussion by Ngib86** [add2cal/add-to-calendar-button#408] [Q&A]
I've used the Example 1: The usual one code for my button but I've asked 2 people to try and add it to their iphone calendar and it doesn't work. It comes up with an error message 'Safari cannot download this file'. 

Any help would be appreciated. Thanks

**Comment by jekuer** [comment]
How are you using the script? Meaning within which environment, what version, etc.

And can you share your code. Except it is the exact same as in the example.
